### DISCUSSION TITLE
web  interface address & DOH conflict

### DISCUSSION THREAD

**Discussion by YBS-PC** [AdguardTeam/AdGuardHome#5974] [General]
I have a vps with an ip for example 12.12.12.12 and a name example.provider.com and the certificate to it. Also on this vps there is a wireguard server with an internal address for example 10.66.77.88. Why if I configure the adguard home web  interface address 10.66.77.88 to have access to the web interface only through wireguard, then my DOH stops working both on wireguard and without it?

**Comment by Freebase394** [comment]
The best way would be to configure using docker on your VPS, that way it would be possible to configure all containers, like your WireGuard or any other personal VPN provider, and all requests have to be made to AdGuardHome
